pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -41,7 +41,10 @@ func (h *Helper) InstallFlags(flags *pflag.FlagSet) {
 
 // GetDockerAuth returns a valid Docker AuthConfiguration entry, and whether it was read
 // from the local dockercfg file
-func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfiguration, bool) {
+func (h *Helper) GetDockerAuth(imageName, runtimeImageName, authType string) (docker.AuthConfiguration, bool) {
+	if runtimeImageName != "" {
+		glog.V(3).Infof("Runtime image detected %s", runtimeImageName)
+	}
```

Comment:
@PI-Victor Why do we need to add this argument? I see that we're just printing it.